docs/framework/unmanaged-api/strong-naming/index.md

```diff
@@ -15,7 +15,7 @@ The strong naming API enables a client to administer strong name signing for ass
  Signing an assembly with a strong name adds a public key encryption to the file containing the assembly manifest. Strong name signing helps verify name uniqueness, prevents name spoofing, and provides callers with a unique identity when a reference is resolved. However, no level of trust is associated with a strong name.  
   
 ## In This Section  
- [Strong Naming Global Static Functions](https://msdn.microsoft.com/library/efa715df-e8cc-48f2-9ec4-26586f0dc8d0)  
+ [Strong Naming Global Static Functions](https://docs.microsoft.com/previous-versions/dotnet/netframework-4.0/aa964998(v=vs.100))  
```

Comment:
given that this is the in this section, all the links that are going to previous-versions should be removed from this listing